client/my-sites/checkout/checkout-thank-you/licensing-thank-you-manual-activation-instructions.tsx

```diff
@@ -65,7 +65,7 @@ const LicensingActivationInstructions: FC< JetpackLicenseKeyProps > = ( {
 				}
 				footerImage={ licensingActivationPluginInstall }
 				showContactUs
-				showProgressIndicator
+				showProgressIndicator={ false }
```

Comment:
@manzoorwanijk for the other products such as the bundles. We actually still display the old flow which the license activation instruction is in a separate page.

should we maybe check here if we are trying to display a standalone installation?

```
showProgressIndicator={ ! jetpackStandaloneProduct }
```

Reply:
That is a nice suggestion. Thanks